src/widgetastic_patternfly4/calendarmonth.py

```diff
@@ -0,0 +1,98 @@
+from selenium.common.exceptions import NoSuchElementException
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import Table
+from widgetastic.widget import TextInput
+from widgetastic.widget import Widget
+
+from .select import Select
+
+
+class BaseCalendarMonth:
+    """Represents calendar month component.
+
+    https://www.patternfly.org/v4/components/calendar-month
+    """
+
+    CALENDAR_HEADER = ".//div[@class='pf-c-calendar-month__header']"
+    MONTH_SELECT_LOCATOR = (
+        f"{CALENDAR_HEADER}//div[contains(@class, 'header-month')]"
+        f"//div[@data-ouia-component-type='PF4/Select']"
+    )
+    YEAR_INPUT_LOCATOR = f"{CALENDAR_HEADER}//div[contains(@class, 'header-year')]/input"
+
+    PREV_BUTTON_LOCATOR = f"{CALENDAR_HEADER}//div[contains(@class, 'prev-month')]"
+    NEXT_BUTTON_LOCATOR = f"{CALENDAR_HEADER}//div[contains(@class, 'next-month')]"
+
+    TABLE = ".//table"
+    SELECTED_DATE_LOCATOR = f"{TABLE}/tbody//td[contains(@class, 'pf-m-selected')]"
+
+    def prev(self):
+        return self.browser.click(self.PREV_BUTTON_LOCATOR)
+
+    def next(self):
+        return self.browser.click(self.NEXT_BUTTON_LOCATOR)
+
+    @property
+    def year(self):
+        el = self.browser.element(self.YEAR_INPUT_LOCATOR)
+        return el.get_attribute("value")
+
+    @year.setter
+    def year(self, value):
+        year_input_widget = TextInput(self, locator=self.YEAR_INPUT_LOCATOR)
```

Comment:
I think better to avoid widget initialization like this.
what do you think if we declare this on top itself?

```
_year = TextInput(locator=<.//input[@....]>)
```